modules/standard/IO.chpl

```diff
@@ -3698,6 +3845,59 @@ inline proc channel.read(ref args ...?k,
   return !error;
 }
 
+proc stringify(args ...?k):string {
```

Comment:
User facing is fine by me!  My first instinct is to call it toString or toStringConcat, but I wouldn't call it a good name